tests/cc/test_looping.cq

```diff
@@ -189,10 +189,6 @@ while (b[0]) {
 #while (1 < i+3) { # Error: Input error: in pass VQ1Asm, phase main: in block 'condComplexExpr': in expression '1 < creg[0] + 3': cannot handle function call within function call 'creg[0] + 3'
 #}
 
-#.condConstExpr
-#while (4==4) {
-#}
-
 # single block, with structured control flow mid way
 # 'while' loop does not seem to be a barrier for scheduling by itself
 .multiKernel
```

Comment:
Jeroen implemented dependence analysis such that there is not automatically a barrier before and after any loop. I haven't reviewed that well enough and don't understand how that could be possible. I would have expected that before and after each loop body (so inside the while) there would be a barrier. This would cater for taking the iteration, i.e. the dependence graph cannot be circular so cannot represent the dependences between two successive iterations.
So, this is not a problem here, but a reminder to investigate this further and to come to this later when that investigation has been done.